src/main/resources/application.properties

```diff
@@ -31,3 +31,5 @@ spring.h2.console.enabled=true
 jwt.header=Authorization
 jwt.secret=secret
```

Comment:
Also... could we change the secret to something secure and very long? We don't need to type it all the time so it shouldn't matter

Reply:
enhanced with commit [b86f9fd](https://github.com/JamHaven/se-carrental-webservice/pull/8/commits/b86f9fd006222d19b72d04962c954907a8d1ec02)

Reply:
das ist sicher so in einem wörterbuch, ich habe an etwas langes, 15 zeichen zufallsgenereiert, gedacht. Länge über Complexity

Reply:
improved complexity and length in commit [6a4aca5](https://github.com/JamHaven/se-carrental-webservice/pull/8/commits/6a4aca5c0eec0951944a0c9b55ccb614553ab793)